net-libs/nghttp2/nghttp2-1.49.0.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# TODO: Add python support.
+
+EAPI=8
+
+inherit autotools multilib-minimal
+
+DESCRIPTION="HTTP/2 C Library"
+HOMEPAGE="https://nghttp2.org/"
+SRC_URI="https://github.com/nghttp2/nghttp2/releases/download/v${PV}/${P}.tar.xz
+	https://dev.gentoo.org/~voyageur/distfiles/${PN}-1.48.0-pthread.patch"
+
+LICENSE="MIT"
+SLOT="0/1.14" # <C++>.<C> SONAMEs
+KEYWORDS="~amd64 ~x86"
```

Comment:
Only thing I fixes is restoring previous keywords, nothing major changed in 1.49 so we can keep the existing ones